v2/pkg/runtime/window.go

```diff
@@ -133,6 +133,24 @@ func WindowUnminimise(ctx context.Context) {
 	appFrontend.WindowUnminimise()
 }
 
```

Comment:
I think we may be missing `IsWindowFullscreen()`

Reply:
I first used the form function isfullscreen.
But I have added the native implementation.